workflows/VGP-assembly-v2/Assembly-Hifi-Trio-phasing-VGP5/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [0.2] 2024-08-12
+
+### Added
+
+- Workflow report
+
+### Changed 
+
+- Exposing the Busco lineage database parameter
+- Fixing a bug happening when merging of assembly statistics
+  
+
+
```

Comment:
```suggestion
- Fix a bug happening when merging assembly statistics

```
It's also nice to describe what the bug is so users know if they need to re-run the workflow